GVFS/GVFS.FunctionalTests/Tools/GVFSProcess.cs

```diff
@@ -94,7 +94,7 @@ private string CallGVFS(string args, bool failOnError = false, string trace = nu
         {
             ProcessStartInfo processInfo = null;
             processInfo = new ProcessStartInfo(this.pathToGVFS);
-            processInfo.Arguments = args + " --internal_use_only_service_name " + GVFSServiceProcess.TestServiceName;
+            processInfo.Arguments = args + " --internal_use_only_parameters " + GVFSHelpers.GetTestServiceParamater();

```

Comment:
Should we (or have we already) pull out the name of the parameter into a constant somewhere?
```csharp
const string InternalUseParametersName = "internal_use_only_parameters";
```